packages/api/tests/moderation-mutewords.test.ts

```diff
@@ -417,7 +495,7 @@ describe(`hasMutedWord`, () => {
           outlineTags: [],
         })
 
-        expect(match).toBe(false)
+        expect(match).toBe(true)
```

Comment:
This update allows `superbad` to match `super-bad`, basically collapses the punctuation. Previously `super bad` would match `super-bad` but not the collapsed version.

This was done so that `blueskys` match `Bluesky's`. I'm trying to think of cases where this would be TOO eager, but haven't come up with anything yet.